velox/functions/sparksql/String.h

```diff
@@ -1181,6 +1181,34 @@ struct FindInSetFunction {
   }
 };
 
+template <typename T>
+struct RepeatFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  static constexpr bool is_default_ascii_behavior = true;
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<Varchar>& result,
+      const arg_type<Varchar>& input,
+      int32_t times) {
+    VELOX_USER_CHECK_LE(times, 10000, "Repeat times is too big.");
+    if (times <= 0) {
+      result.resize(0);
+      return;
+    }
+    auto inputSize = input.size();
+    if (inputSize == 0) {
+      result.resize(0);
+      return;
+    }
```

Comment:
It seems the two branches could be combined as below.

> if (times <= 0 || input.size() == 0) {
result.resize(0);
return;
}